uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/request-ctrl.js

```diff
@@ -977,30 +971,25 @@ define([
       }
 
       /**
-       * Loads just one pre-selected managee
-       *
-       * @return {Promise}
-       */
-      function loadManagee () {
-        return Contact.find(this.directiveOptions.selectedContactId)
-          .then(function (contact) {
-            this.managedContacts = [contact];
-          }.bind(this));
-      }
-
-      /**
        * Loads the managees of currently logged in user
        *
        * @return {Promise}
        */
       function loadManagees () {
-        return Contact.find(this.directiveOptions.contactId)
-          .then(function (contact) {
-            return contact.leaveManagees();
-          })
-          .then(function (contacts) {
-            this.managedContacts = contacts;
-          }.bind(this));
+        if (this.directiveOptions.selectedContactId) {
```

Comment:
@AkA84 Thank you. Could you explain why do we need to fetch full list if we don't need full list in the end? Also, why do we need to make 2 calls instead of one?